test/e2e/service.go

```diff
@@ -567,10 +577,12 @@ var _ = Describe("Services", func() {
 		jig.TestReachableUDP(nodeIP, udpNodePort, kubeProxyLagTimeout)
 
 		By("hitting the TCP service's LoadBalancer")
-		jig.TestReachableHTTP(tcpIngressIP, svcPort, loadBalancerLagTimeout)
+		jig.TestReachableHTTP(tcpIngressIP, svcPort, loadBalancerCreateTimeout)
```

Comment:
Ah, my motivation here was that AWS was timing out waiting for the LB to be ready; it would not be surprising if the ELB took 5 minutes to start serving traffic after it was created in the API.  It might be we simply have to bump up LagTimeout on AWS.

I'll clean this up...